config/config.go

```diff
@@ -4,13 +4,13 @@ import "github.com/kelseyhightower/envconfig"
 
 // Specification config struct
 type Specification struct {
-	Port           string `default:"8080"`
-	Polling        int    `default:"43200"` // 43200 seconds = 12 hours
-	VersionsAPIURL string
-	DoctorAPIURL   string
-	APIVersion     string
-	CheckVersions  bool   `default:"true"`
-	DeisNamespace  string `default:"deis"`
+	Port           string `default:"8080" envconfig:"PORT"`
+	Polling        int    `default:"43200" envconfig:"POLL_INTERVAL_SEC"` // 43200 seconds = 12 hours
+	VersionsAPIURL string `envconfig:"VERSIONS_API_URL"`
+	DoctorAPIURL   string `envconfig:"DOCTOR_API_URL"`
+	APIVersion     string `envconfig:"API_VERSION"`
+	CheckVersions  bool   `default:"true" envconfig:"CHECK_VERSIONS"`
+	DeisNamespace  string `default:"deis" envconfig:"CHECK_VERSIONS"`
```

Comment:
fixed in https://github.com/deis/workflow-manager/pull/50/commits/1789e75f9986a484612071cde9eb079dc2963241